src/services/users.services.js

```diff
@@ -0,0 +1,61 @@
+import axios from "axios";
+import { API_URL } from "../config.js";
+
+const BASE_URL = API_URL+"/users/";
+
+export const getUsersAmount = async (token) => {
+  try {
+      const res = await axios.get(BASE_URL+"amount", {
+        headers: { Authorization: `Bearer ${token}` },
+      });
+      return res;
+  } catch (error) {
+      console.log("Error ", error);
+  }
+}
+
+export const fetchUsers = async (token, page) => {
+  try {
+    const res = await axios.get(BASE_URL+"?size=9&page="+page, {
+      headers: { Authorization: `Bearer ${token}` },
+    });
+    return res.data;
+  } catch (error) {
+    throw new Error('error fetcheando usuario');
+  }
+};
+
+export const createUser = async (user, token) => {
+  try {
+    const { data } = await axios.post(BASE_URL, user, {
+      headers: { Authorization: `Bearer ${token}` },
+    },);
+    return data;
+  } catch (e) {
+    console.log(e.message);
+    throw new Error('error creando usuario');
```

Comment:
```suggestion
    throw new Error(e.message);
```
Algo asi seria mejor?, tambien me preocupa que NO se este mostrando el error que venga del servidor, es decir el que esta en "response.message"

Reply:
Y esta es una correcion no solo para este servicio si no para todos

Reply:
Igual si es mucho trabajo me dicen y la podemos dejar para una tarea aparte

Reply:
Todavia no esta resuelto en el PR, estamos en proceso de implementar los toasts para los errores del try y catch.